libkbfs/kbfs_ops.go

```diff
@@ -678,7 +678,7 @@ func (fs *KBFSOpsStandard) Status(ctx context.Context) (
 		err := fillInJournalStatusUnflushedPaths(
 			ctx, fs.config, jServerStatus, tlfIDs)
 		if err != nil {
-			return KBFSStatus{}, nil, err
+			return KBFSStatus{}, ch, err
```

Comment:
Maybe add a comment explaining that even `err` is `nil` the caller in `remote_status.go` still relies on `ch`?